USAGE.md

```diff
@@ -1,370 +1,707 @@
+The following recipe retrieves an object according to its name:
 
 ```python
-def get_technique_by_group(src, stix_id):
-    relations = src.relationships(stix_id, 'uses', source_only=True)
-    return src.query([
+def get_technique_by_name(thesrc, name):
+    filt = [
         Filter('type', '=', 'attack-pattern'),
-        Filter('id', 'in', [r.target_ref for r in relations])
-    ])
-
-group = get_group_by_alias(fs, 'Cozy Bear')[0]
-get_technique_by_group(fs, group)
+        Filter('name', '=', name)
+    ]
+    return thesrc.query(filt)
+# get the technique titled "System Information Discovery"
+get_technique_by_name(src, 'System Information Discovery')
 ```
 
-### Get all Techniques used By a Group's Software
-This example is fairly complex.  First we must query the group's relationships to find the software that the group uses.  Then we must use the relationships for each piece of software in order to find the techniques.  Notice the expression for `software_uses` involves directly filtering the relationship objects rather than using the `relationships` method of the DataSource `src`.  This is because, as of this writing, calls to `query` are expensive, and it is better to minimize them.
+### By Alias
+The following methodology can be used to find the group corresponding to a given alias:
 
 ```python
-from stix2.utils import get_type_from_id
+def get_group_by_alias(thesrc, alias):
+    return thesrc.query([
+        Filter('type', '=', 'intrusion-set'),
+        Filter('aliases', '=', alias)
+    ])[0]
+    
+get_group_by_alias(src, 'Cozy Bear')
+```
 
-def get_techniques_by_group_software(src, group_stix_id):
-    # get the malware, tools that the group uses
-    group_uses = [
-        r for r in src.relationships(group_stix_id, 'uses', source_only=True)
-        if get_type_from_id(r.target_ref) in ['malware', 'tool']
-    ]
 
-    # get the technique stix ids that the malware, tools use
-    software_uses = src.query([
-        Filter('type', '=', 'relationship'),
-        Filter('relationship_type', '=', 'uses'),
-        Filter('source_ref', 'in', [r.source_ref for r in group_uses])
-    ])
 
-    #get the techniques themselves
-    return src.query([
-        Filter('type', '=', 'attack-pattern'),
-        Filter('id', 'in', [r.target_ref for r in software_uses])
-    ])
+## Getting multiple objects
+The recipes in this section address how to query the dataset for multiple objects.
 
-group = get_group_by_alias(fs, 'Cozy Bear')[0]
-get_techniques_by_group_software(fs, group)
-```
+When working with queries to return objects based on a set of characteristics, it is likely that you'll end up with a few objects which are no longer maintained by ATT&CK. We keep these outdated objects around so that workflows depending on them don't break, but we recommend you avoid using them when possible. Please see the section [working with deprecated and revoked objects](#Working-with-deprecated-and-revoked-objects) for more information.
```

Comment:
1. It would be nice to set this in a box or something like that since it's a warning to users. That would make the intent clearer.
2. I assume that "outdated objects" == "deprecated and revoked objects" but that confused me at first. Assuming that I have that right, one way to help make this clearer would be to add a sentence after the first sentence "These are objects marked as deprecated or revoked." Something like that would make the meaning in the rest of the paragraph more obvious.

Reply:
1. Not sure if github flavored markdown will let me do a warning box, but I'll see what I can do.
2. good call, I'll add the sentence.

Reply:
For 1, the best I can do without barbaric workarounds like the following (using a code block with `diff` formatting) 
```diff
- test in red
+ test in green
! test in orange
# test in gray
``` 
is to use a warning icon like so:

![image](https://user-images.githubusercontent.com/28868888/85875134-c28f2480-b7a1-11ea-8fef-9304fceb7e0a.png)